tendrl/gluster_integration/sds_sync/brick_device_details.py

```diff
@@ -0,0 +1,92 @@
+from tendrl.commons.event import Event
+from tendrl.commons.message import Message
+from tendrl.commons.utils import cmd_utils
+
+
+def get_brick_source_and_mount(brick_path):
+    # source and target correspond to fields "Filesystem" and
+    # "Mounted on" from df command output. The below command
+    # gives the filesystem and mount point for a given path,
+    # Eg: "/dev/mapper/tendrlMyBrick4_vg-tendrlMyBrick4_lv " \
+    #     "/tendrl_gluster_bricks/MyBrick4_mount"
+
+    command = "df --output=source,target " + brick_path + " |sed 1d"
+    cmd = cmd_utils.Command(command)
+    out, err, rc = cmd.run()
+    if rc != 0:
+        Event(
```

Comment:
@nnDarshan Why are we not using blivet.util.get_mount_device(mountpoint) vs the "df" cmd

Also, use log_utils